File: UIPresentation/UI/src/app/models/tandem-models/summary-external.ts
import { Time } from "@angular/common"
import { StatusEnum } from "src/app/enums/status-enum.enum"
export class SummaryExternal {
    requestNumber!:string
    corporateCode!:number
    status!:StatusEnum
    uploadDate!:Date
    quantity!:number
    amount!:number
    registrationNumber!:number
    systemEnteredDate!:Date
    systemEnteredTime!:Time
    systemEnteredRegistrationNumber!:number
    isChecked!:boolean
}

File: Tandem/Dtos/TandemDtos/TandemPaymentInquiryExternalDtos/SummaryExternal.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Text.Json.Serialization;
using System.Threading.Tasks;
using Entities.Concrete.Enums;

namespace Tandem.Dtos.TandemDtos.TandemPaymentInquiryExternalDtos
{
    public class SummaryExternal
    {
        public string RequestNumber { get; set; }
        public int CorporateCode { get; set; }
        public StatusEnum Status { get; set; }
        public DateTime UploadDate { get; set; }
        public int Quantity { get; set; }
        public decimal Amount { get; set; }
        public int RegistrationNumber { get; set; }
        public DateTime SystemEnteredDate { get; set; }
        public TimeSpan SystemEnteredTime { get; set; }
       public int SystemEnteredRegistrationNumber { get; set; }
    }
}
